### ISSUE TITLE
Anti-neurochondrin antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10547]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against neurochondrin 

 PMID: 36437687 Cerebellar ataxia associated with anti-neurochondrin antibodies has only been described in 19 cases; however, the number of unrecognized PACAs (primary autoimmune cerebellar ataxia ) may be higher. As anti-neurochondrin antibodies target an intracellular antigen and exhibit a mainly cytotoxic T-cell-mediated pathogenesis, important therapeutic implications may result. Because of the severe and rapid clinical progression, aggressive immunotherapy was warranted. This case highlights the need for rapid diagnosis and therapy in PACA, as stabilization and even improvement of symptoms are attainable. 

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624259 Non-specific autoimmune cerebellar ataxia with characteristic antibodies

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000880
- label: Anti-neurochondrin antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:36437687
- def: The presence of autoantibodies in the serum that react against neurochondrin .
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10547
- synonyms:
